frontend/test/lit/test_multi_qubit_gates.py

```diff
@@ -62,3 +62,17 @@ def circuit():
 
 
 print(circuit.mlir)
+
+
+# CHECK-LABEL: public @jit_circuit
+@qjit(target="mlir")
+@qml.qnode(qml.device("braket.local.qubit", wires=2, shots=100))
+def circuit(x: float):
+    # CHECK: {{%.+}} = quantum.custom "ISWAP"() {{.+}} : !quantum.bit, !quantum.bit
```

Comment:
@paul0403 could you address these codefactor suggestions?

Reply:
https://github.com/PennyLaneAI/catalyst/pull/458/checks?check_run_id=20880970204

Reply:
This added function is in accordance with the style of the rest of the file frontend/test/lit/test_multi_qubit_gates.py. They are all named "circuit" and don't have docstrings.

Reply:
we should likely fix them all sometime :t-rex:  don't worry about these codefactor CI suggestions there! This is not a strong merge requirement